src/Microsoft.Health.Fhir.Shared.Web/Startup.cs

```diff
@@ -113,6 +115,16 @@ private void AddTaskHostingService(IServiceCollection services)
                 .AsSelf()
                 .AsImplementedInterfaces();
             services.Configure<TaskHostingConfiguration>(options => Configuration.GetSection("TaskHosting").Bind(options));
+
+            IEnumerable<TypeRegistrationBuilder> jobs = services.TypesInSameAssemblyAs<ImportOrchestratorJob>()
```

Comment:
How does this TypesInSameAssemblyAs method work? I want to see how easy it will be for future jobs to be automatically picked up.

Reply:
Any IJob added in the Core assembly (which is where ImportOrchestratorJob is) should be automatically registered. Is we add jobs from other assemblies we'll need to amend the registration